CAIPs/caip-261.md

```diff
@@ -0,0 +1,384 @@
+Consequently, the most recent assertion effectively replaces the older ones in the context of the trust graph, ensuring that the trust information remains current and accurate.
+
+#### Trust Revocation
+When an issuer decides to revoke a previously issued trust assertion, they should generate a new assertion that includes a 'credentialStatus' attribute, with a 'statusPurpose' set to 'revocation'. 
+This new assertion should explicitly reference the assertion being revoked. 
+By doing so, the system and its participants can easily identify the revocation status of any assertion, ensuring the trust graph remains accurate and up-to-date with the current trust relationships.
+
+**Revocation of trust assertion:**
+```json
+"@context": ["https://www.w3.org/2018/credentials/v1"],
+"type": ["VerifiableCredential", "PeerTrustCredential"],
+"issuanceDate": "2024-02-29T14:31:56.273Z",
+"issuer": "did:pkh:eip155:1:0x44dc4E3309B80eF7aBf41C7D0a68F0337a88F044",
+"credentialStatus": {
+    "id": "ipfs://QmcwYEnWysTyepjjtJw19oTDwuiopbCDbEcCuprCBiL7gt",
+    "type": "CredentialStatus",
+    "statusPurpose": "revocation",
+},
+```
+#### Trust Graph Expension
+##### Implicit Trust Assertions
+While the Peer Trust Assertion model outlined previously focuses on explicit trust signals, the trust graph can be further enriched with additional implicit on-chain and/or off-chain trust signals related to its peers. These can include Proof of Humanity, Proof of Membership, Proof of Contributions, Proof of Attendance, Social Graphs, among others. 
+This standard does not define the handling of implicit trust; it leaves the integration of relevant data to the discretion of the final consumer, depending on the specific use-case. 
+
+##### Additional Explicit Trust Assertions
+The trust graph can also be enhanced with additional explicit trust signals derived from any model, broadening the scope of trust assessment.
+
+### Trust Assertions Verification
+The standard presumes that both the `issuer` property will be dereferenced and the complete contents of the `credentialSubject` will be consumed only after the wire-formats and signed-envelopes have been verified.
+
+#### Signature Verification
+
+The veracity and integrity of trust assertions are paramount to ensuring unbiased insights. 
+All trust assertions SHOULD be cryptographically signed by the issuer using strong cryptographic methods and verified prior to consumption. 
+The standard supports any strong signature methods, such as: ECDSA, EdDSA, Schnorr Signatures, RSA...
+
+EIP-712 should be considered a complementary cryptographic proof method alongside others like ECDSA, EdDSA, Schnorr, and RSA for on-chain verifiable credentials. 
+Its inclusion emphasizes the importance of user-friendly, secure, and efficient interactions with blockchain-based identity and credential systems.
+It is noteworthy that EIP-712 mandates the presence of all fields, even if some are left empty in order to enable their verification.
+

```

Comment:
This example looks a little iffy? https://w3c-ccg.github.io/ethereum-eip712-signature-2021-spec/ makes it sound like there are some mandatory fields missing here?